src/openforms/emails/templates/emails/cosign.html

```diff
@@ -0,0 +1,25 @@
+{% load static i18n %}
+
+<p>
+    {% blocktranslate %}
+        This is a request to co-sign form "{{ form_name }}".<br />
```

Comment:
Does this use the right language of the form name?

Reply:
I tested this by overriding `LANGUAGE_CODE` and it works. I hope this is what you mean. Maybe I should another test for this as well?